src/store/createStore.ts

```diff
@@ -16,22 +19,57 @@ export interface Store {
     update: (value: string) => void;
     reset: () => void;
   };
+
+  // Explore Endpoints
+  exploreEndpoints: {
+    network: Network | EmptyObj;
```

Comment:
can we have a different network within `exploreEndpoints` than the top level one?

Reply:
They should be in sync most of the time, except when the top level changes. If they are different, we know that the top level network changed, so we need to clear the current endpoint data from the store.